add explanation of working in a text editor and how to run files in p… (#424)
codebar/tutorials: python/lesson1/tutorial.md

In this tutorial, we're going to start to work with the basic types of Python:
strings (for text) and integers & floats (for numeric values).

> Notice that in this tutorial you are working in REPL (IDLE). You find more information

Reviewer: ```suggestion
Notice that in this tutorial you are working in REPL (IDLE). You can find more information
```

    >>> print('The result of 2 + 2 is', 2 + 2)

### Saving Your Work

In this tutorial you coded in the REPL (IDLE), but a lot of times you want

> to save your code instead. In these casees you would use a text editor. 

Reviewer: ```suggestion
to save your code instead. In such cases you can save your code to a file using a text editor. 
```

We give some information on text editors in our [Getting started guide](general/setup/tutorial.html) .

Open your text editor and write the code from the first exercise:

    print("Hello, World!")

Save the file as `ex1.py`. You can name your files as you like, 
but they should end with `.py`, so python can read them easily.
Reading your file into Python you will use your `cmd` or `terminal` shell again. 
You can read your file with the following command (type without the $ sign):

	$ python ex1.py

If you have `>>>`in front of your code, you are still in REPL(IDLE) and need to exit
it with:

	>>> quit()

> Than you should be able to load the file. 

Reviewer: ```suggestion
Then you should be able to load the file. 
```
